examples/non-fungible-token/src/lib.rs

```diff
@@ -0,0 +1,473 @@
+// Fungible Token Smart Contract.
+// Implementation based on https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol
```

Comment:
typo?
meant that? https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC721/ERC721.sol